bench/llvm/optimized/MCObjectStreamer.ll

```diff
@@ -1206,13 +1207,15 @@ switch.lookup:                                    ; preds = %4
   %43 = getelementptr inbounds nuw i8, ptr %9, i64 48
   %44 = load i64, ptr %43, align 8, !tbaa !301
   %45 = trunc i64 %44 to i32
-  %46 = call range(i32 0, 33) i32 @llvm.cttz.i32(i32 %2, i1 true)
-  %switch.offset = add nuw nsw i32 %46, 1
+  %switch.tableidx = add nsw i32 %2, -1
+  %46 = sext i32 %switch.tableidx to i64
+  %switch.gep = getelementptr inbounds [8 x i32], ptr @switch.table._ZN4llvm16MCObjectStreamer13emitValueImplEPKNS_6MCExprEjNS_5SMLocE, i64 0, i64 %46
+  %switch.load = load i32, ptr %switch.gep, align 4
```

Comment:
Regression for the case where switch maps to the cttz result. This now goes through the lookup table.

Reply:
Ugh, might imply we should tune the option in the pipeline further?